### PR TITLE
chore: add README

### LINKED ISSUE #14: Create README file
@alequetzalli mentioned that it would be great to include a README file in this repo so that we can better introduce the repo's purpose to the community & provide instructions on how to contribute. Any ideas for the structure of this file are welcome!

### PR DESCRIPTION
**Pull request by mcturco**
Adding README file as suggested by @alequetzalli 🙌 

If there is anything I should tweak let me know!

(Resolves #14)

### CODE REVIEW

**Comment by mcturco** [comment]
Also - looking at the README in preview mode within the commit and for some reason the image is showing up as broken, but it is linked correctly so I'm not sure why! Here is the preview from VS Code

Screen Shot 2021-12-15 at 5 04 22 PM

**Comment by alequetzalli** [comment]
Hey there! The first thing I would recommend is that you remove all of the unnecessary HTML tags you added because this file is a markdown file so you don't need to put those. 😄 are you familiar with markdown syntax?

https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
+<div>
+<a href="https://www.asyncapi.com"><img src="./assets/readme-banner.svg" alt="AsyncAPI Brand Banner" width="100%"></a>
+</div>
+<br>
+<h2>Welcome to the brand ✨</h2>
+<p>Hi there! Thanks for stopping by Emoji 😄 In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable. Before you submit your contribution, please read our <a href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md" target="_blank">Code of Conduct</a> and <a href="https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi" target="_blank">Contribution Guidelines</a>.</p>
```
Comment (alequetzalli) [inline README.md:7]:
```suggestion
<p>Hi there! Thanks for stopping by 😄 In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable. Before you submit your contribution, please read our <a href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md" target="_blank">Code of Conduct</a> and <a href="https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi" target="_blank">Contribution Guidelines</a>.</p>
```

**Comment by alequetzalli** [comment]
> Also - looking at the README in preview mode within the commit and for some reason the image is showing up as broken, but it is linked correctly so I'm not sure why! Here is the preview from VS Code
> 
> 
> 
> Screen Shot 2021-12-15 at 5 04 22 PM
> 
> 

As part of the changes you need to make to remove the HTML tags and use only markdown syntax, I would go ahead and use the markdown syntax for images and see if it's still giving you that broken issue 🙂✌🏽

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+</tr>
+<tr>
+<td align="middle">9</td>
+<td>Feedback and iteration on design system and UI Kit</td>
+</tr>
+</tbody>
+</table>
+<br>
+<h2>Join the discussion</h2>
+<p>Join the <a href="https://asyncapi.com/slack-invite" target="_blank">AsyncAPI Slack workspace</a> and the <code>#12_design</code> channel to follow all conversations relating to this repo as well as other topics of design.</p>
```
Comment (alequetzalli) [inline README.md:73]: Do you want to add a `Design` category in our GH Discussions? 
https://github.com/asyncapi/community/discussions 

Screen Shot 2021-12-15 at 3 30 18 PM

**Review by alequetzalli — CHANGES_REQUESTED** [review]

**Comment by mcturco** [comment]
Hey @alequetzalli! Thanks! Yeah I was wondering if I should have only used the mark down syntax but I was referencing the [README in the community repo](https://github.com/asyncapi/community/blob/master/README.md?plain=1) and I saw HTML tags so I thought it would be okay!

No worries though, its probably better anyway if I use the correct syntax. I will work on that now and then tag you again when its ready for review! 😄

**Comment by mcturco** [comment]
@alequetzalli okay! We should be all set now, I updated the format to markdown only 😄

**Comment by alequetzalli** [comment]
> Hey @alequetzalli! Thanks! Yeah I was wondering if I should have only used the mark down syntax but I was referencing the [README in the community repo](https://github.com/asyncapi/community/blob/master/README.md?plain=1) and I saw HTML tags so I thought it would be okay!


Ahhh I know what you mean... not to throw anyone under the bus but all of those HTML tags for the images and sponsors in the community repo README was made by Lukasz 😂... when I edited that file I only improved it and didn't really take the time to look at fixing that stuff.

I think a good README example is the Initiative's README, from what I remember that one was completely rewritten and cleaned out, so it should only have proper markdown syntax 🙂👍🏽

**Comment by mcturco** [comment]
@alequetzalli 

> I think a good README example is the Initiative's README, from what I remember that one was completely rewritten and cleaned out, so it should only have proper markdown syntax 🙂👍🏽

Definitely! Yeah I started looking at that one this morning when I was changing up the syntax. I agree with you that the markdown syntax is MUCH cleaner in this case!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+[![AsyncAPI Brand Banner](./assets/readme-banner.svg)](https://www.asyncapi.com)
+
+## Welcome to the brand ✨
```
Comment (alequetzalli) [inline README.md:4]:
```suggestion
## Welcome to the AsyncAPI brand ✨
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Hi there! Thanks for stopping by 😄 In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable. Before you submit your contribution, please read our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi).
```
Comment (alequetzalli) [inline README.md:5]:
```suggestion
Hi there! Thanks for stopping by. 😄 In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable. Before you submit your contribution, please read our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi).
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- Mascot ideation/illustration
+- UI Kit
+
+Issues in this repo are open to all who would like to give feedback and/or provide visual work.
+
+## Roadmap 🚗 💨
+View the chart below to learn about the current goals for the AsyncAPI brand!
+
+|  Step  |                   Goal                     |
+| :----: | ------------------------------------------ |
```
Comment (alequetzalli) [inline README.md:25]: For consistency, I would prob pick `&` or `and` throughout. 😀👍🏽

**Review by alequetzalli — COMMENTED** [review]
Just 2 typos and 1 recommendation :) 

The rest LGTM 🚀

**Comment by mcturco** [comment]
@alequetzalli thank you! Just committed all your suggestions. LMK if it looks all good!

**Review by alequetzalli — APPROVED** [review]
/rtm

**Review by alequetzalli — APPROVED** [review]

**Comment by alequetzalli** [comment]
Screen Shot 2021-12-16 at 10 12 25 AM

@mcturco  Hmmm... I think something might be off in the settings for this PR? the /rtm script that Lukasz made doesn't seem to work.. and also there is a note that states `you are not authorized to merge` 

what do you think, missy? 🤔

**Review by fmvilas — APPROVED** [review]
:shipit:

**Comment by fmvilas** [comment]
Only @mcturco and I are code owners of this repo so that's why the /rtm command is not working. I first have to approve.

**Pull request merged**